api/hearings/index.ts

```diff
@@ -126,3 +126,18 @@ export async function updateHearingActuals(req: EnhancedRequest, res: Response,
     next(error);
   }
 }
+
+/**
+ * submitHearingActuals - submit hearing actuals
+ */
+export async function submitHearingActuals(req: EnhancedRequest, res: Response, next: NextFunction) {
+  const reqBody = req.body;
```

Comment:
reqBody couldn't set as null as per API contract.

Reply:
@rayliangatos - as discussed I have updated the code to use null.